bciers/apps/reporting/src/app/components/additionalInformation/additionalReportingData/AdditionalReportingData.tsx

```diff
@@ -2,22 +2,68 @@ import { Suspense } from "react";
 import Loading from "@bciers/components/loading/SkeletonForm";
 import { getRegistrationPurpose } from "@reporting/src/app/utils/getRegistrationPurpose";
 import AdditionalReportingDataForm from "@reporting/src/app/components/additionalInformation/additionalReportingData/AdditionalReportingDataForm";
+import { getReportAdditionalData } from "@reporting/src/app/utils/getReportAdditionalData";
+
+const REGULATED_OPERATION = "OBPS Regulated Operation";
+const NEW_ENTRANT = "New Entrant Operation";
+const CAPTURE_TYPES = {
+  ON_SITE_USE: "On-site use",
+  ON_SITE_SEQUESTRATION: "On-site sequestration",
+  OFF_SITE_TRANSFER: "Off-site transfer",
+};
+
+interface AdditionalReportingDataProps {
+  versionId: number;
+}
+const transformToFormData = (
+  data: {
+    capture_emissions: any;
+    emissions_on_site_use: any;
+    emissions_on_site_sequestration: any;
+    emissions_off_site_transfer: any;
+    electricity_generated: any;
+  },
+  registrationPurpose: string,
+) => ({
+  captured_emissions_section: {
+    capture_emissions: data.capture_emissions,
+    capture_type: [
+      data.emissions_on_site_use && CAPTURE_TYPES.ON_SITE_USE,
+      data.emissions_on_site_sequestration &&
+        CAPTURE_TYPES.ON_SITE_SEQUESTRATION,
+      data.emissions_off_site_transfer && CAPTURE_TYPES.OFF_SITE_TRANSFER,
+    ].filter(Boolean),
```

Comment:
I'm not sure what this is supposed to do, but there's a few issues I can spot:
- `data.emissions_on_site_use` will evaluate to `false` if the value is present but 0 or ""
- while `Boolean` is the constructor for the object type derived from `boolean` and a function (so it doesn't trip up typescript) it won't work as an array filter predicate